### PR TITLE
Fix a few minor bugs

### PR DESCRIPTION
**Pull request by llucax**
* Fix wrong optional dependency name
* Label docs/*.py as tooling
* Fix trailing space in template
* Add missing dependency for mkdocs

### CODE REVIEW

File: cookiecutter/{{cookiecutter.github_repo_name}}/.github/RELEASE_NOTES.template.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 ## Upgrading
 
-<!-- Here goes notes on how to upgrade from previous versions, including deprecations and what they should be replaced with --> 
+<!-- Here goes notes on how to upgrade from previous versions, including deprecations and what they should be replaced with -->
```
Comment (Marenz) [inline cookiecutter/{{cookiecutter.github_repo_name}}/.github/RELEASE_NOTES.template.md:9]: "Here goes notes" reads a bit weird to me..

**Review by Marenz — APPROVED** [review]

File: cookiecutter/{{cookiecutter.github_repo_name}}/.github/RELEASE_NOTES.template.md (hunk 1 above)
Comment (shsms) [reply cookiecutter/{{cookiecutter.github_repo_name}}/.github/RELEASE_NOTES.template.md:9]: "Here go notes"? :D

Comment (llucax) [reply cookiecutter/{{cookiecutter.github_repo_name}}/.github/RELEASE_NOTES.template.md:9]: Well, automerge was enabled, so maybe open an issue for it?

Comment (Marenz) [reply cookiecutter/{{cookiecutter.github_repo_name}}/.github/RELEASE_NOTES.template.md:9]: "here go the notes on.."?

Comment (shsms) [reply cookiecutter/{{cookiecutter.github_repo_name}}/.github/RELEASE_NOTES.template.md:9]: I also feel "here" and "go" don't go well together.  It has to be "Here come the notes on..." or "There go the notes on..."

Comment (Marenz) [reply cookiecutter/{{cookiecutter.github_repo_name}}/.github/RELEASE_NOTES.template.md:9]: How about just "Notes on how to upgrade.."?

Comment (shsms) [reply cookiecutter/{{cookiecutter.github_repo_name}}/.github/RELEASE_NOTES.template.md:9]: That was the first good one.  How about "Add notes here, on ..."?

Comment (Marenz) [reply cookiecutter/{{cookiecutter.github_repo_name}}/.github/RELEASE_NOTES.template.md:9]: Also good, yes

**Pull request merged**